testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -56,10 +56,10 @@
 import org.intellij.lang.annotations.Language;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
```

Comment:
Separate commit.

Also, just noticed that one of the overrides `BaseConnectorTest#ensureTestNamingConvention`, does not have `@Test` annotation (actually it has testng's `@Test`, but not Junit's `@Test`).
As a result, the override won't be called.
Please update `BaseConnectorTest.ensureTestNamingConvention` to have additional junit `@Test` annotation (but keep testng's for now, since all other test methods are testng's)

Reply:
When extracting the commit please be so nice to include message from https://github.com/trinodb/trino/commit/6858091dfda01c309555182c7dc5694256c951ae

> For many test classes, this it the last method that is not converted.
For example, running `TestInformationSchemaConnector` with TestNG will
run only this method. Moving this will make tests like
`TestInformationSchemaConnector` JUnit-only and improve test overall
time (fewer `createQueryRunner` invocations).
> 
> Obviously, it can be the first JUnit test method for some other classes,
but
> 
> - the train cannot be stopped. We do this sooner or later.
> - `AbstractTestQueries` are already JUnit; `BaseConnectorTest` extends
  from `AbstractTestQueries` and `BaseConnectorTest` are most likely to
most expensive setups, because they create dependant services.

Reply:
In which order I should add this one? Because in either one one of the commits will be broken on it's own